app/tools/android/make_apk.py

```diff
@@ -142,12 +140,30 @@ def FindExtensionJars(root_path):
         extension_jars.append(extension_jar)
   return extension_jars
 
+# Try to parse out suitable app_versionCodeBase based on app_version,
+# app_version "xx.xx.xxx" will generate app_versionCodeBase "xxxxxxx"
+# For example,  "1.2.3" will generate app_versionCodeBase "0102003"
+# "1.2" will generate "0001002"
```

Comment:
Should not `1.2` be converted to `010200`? Otherwise 1.2.1(010201) will be bigger than 2.1(000201)